aries-site/src/pages/foundation/typography.mdx

```diff
@@ -9,9 +9,9 @@ import { Example } from '../../layouts';
 
 Our typography is about more than words. It’s an integral part of our personality and design. When we’re making a statement, we want our visual language to be clear, recognized and understood.
 
-## MetricHPE styles
+## Graphik styles
```

Comment:
should this be `"HPE Graphik"` follow up with @oliverHPE

Reply:
okay did you follow up with @oliverHPE on if we should of been using `"HPE Graphik"`